packages/client/src/admin/Group/Form.tsx

```diff
@@ -190,6 +190,9 @@ function GroupForm(props: Props) {
   const [groupAdmins, setGroupAdmins] = useState('');
   const initialValue: GroupInput = props.initialValue || defaultGroupValue;
   const submitText = type === 'update' ? 'Update' : 'Create';
+  if (isNil(initialValue.launchGroupOnly)) {
```

Comment:
I thought you should add initial value from upstream instead of adding check `isNil` here.

https://github.com/InfuseAI/primehub-console/blob/d86c3dc829fc6e3c834f0e1bf72cf7ef8fb53785/packages/client/src/admin/Group/GroupUpdate.tsx#L246-L252

Reply:
The initialValue of  `group`  is supposed to be the result of graphql query.  So i prefer not to change the value at the upstream.

`initialValue.launchGroupOnly` is reasonable input if the user have not enabled the **Shared Volume** yet.